config/adapters_config.yaml

```diff
@@ -554,4 +554,32 @@ dbsuper_super_enhancer_regulates_gene:
 
   outdir: dbsuper
   nodes: False
+  edges: True
+
+enhancer_atlas_enhancer:
+  adapter:
+    module: biocypher_metta.adapters.enhancer_atlas_adapter
+    cls: EnhancerAtlasAdapter
+    args:
+      enhancer_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer.bed.gz
+      enhancer_gene_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer_gene
+      tissue_to_ontology_filepath: ./aux_files/enhancer_atlas_tissues_to_ontology.pkl
+  
+  outdir: enhancer_atlas
+  nodes: True
+  edges: False
+
+enhancer_atlas_enhancer_regulates:
+  adapter:
+    module: biocypher_metta.adapters.enhancer_atlas_adapter
+    cls: EnhancerAtlasAdapter
+    args:
+      enhancer_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer.bed.gz
+      enhancer_gene_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer_gene
+      tissue_to_ontology_filepath: ./aux_files/enhancer_atlas_tissues_to_ontology.pkl
+      type: enhancer to gene association
+      label: enhancer_gene
```

Comment:
is there an `enhancer_gene` definition in the schema? Also shouldn't label be `regulates` ?

Reply:
it is `input_label` and the output label is `regulates`

Reply:
I've changed `label` to `input_label`